queries/powershell/highlights.scm

```diff
@@ -0,0 +1,323 @@
+; Punctuation
+;------------
+[
+  "("
+  ")"
+  "["
+  "]"
+  "{"
+  "}"
+] @punctuation.bracket
+
+(hash_literal_expression
+  [
+    "@{"
+    "}"
+  ] @punctuation.special)
+
+(array_expression
+  [
+    "@("
+    ")"
+  ] @punctuation.special)
+
+(member_access
+  "." @punctuation.special)
+
+(pipeline
+  "|" @punctuation.special)
```

Comment:
Personally I think this fits more as `@operator` (also the `pipeline` parent qualification is unnecessary because it can only be found within a `pipeline` node)